proto/tabletmanagerdata.proto

```diff
@@ -333,6 +334,15 @@ message ConcludeTransactionRequest {
 message ConcludeTransactionResponse {
 }
 
+
+message MysqlSystemMetricsRequest {
```

Comment:
IMO, `MySQLSystem` is unclear. To me, that means MySQL metrics. IMO `MySQLHost` is more accurate and clear. Since these are all metrics from the OS/host rather than MySQL itself right?

Reply:
Changes to `MySQLHost*` naming scheme.